dockerfile-prod

```diff
@@ -0,0 +1,15 @@
+# build environment
+FROM node:12.2.0-alpine as build
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+COPY package.json /app/package.json
+RUN npm install --silent
+RUN npm install @vue/cli@3.7.0 -g
```

Comment:
@vue/cli hoeft volgens mij niet globaal geinstalleerd te worden omdat de cli-service als dependency al in het project zit.

Reply:
Klopt.